exonum-java-binding-core/rust/src/runtime/cmd.rs

```diff
@@ -7,7 +7,7 @@ use exonum::node::NodeConfig;
 use failure;
 use toml::Value;
 
-const EJB_DEBUG: &str = "EJB_DEBUG";
+const EJB_USER_PARAMETERS: &str = "EJB_USER_PARAMETERS";
```

Comment:
JVM_ARGUMENTS?
(EJB_USER_PARAMETERS suggests that these are parameters of EJB).